service/model/project.go

```diff
@@ -50,21 +50,21 @@ type UpdateProjectInput struct {
 }
 
 type ReleaseNotificationConfig struct {
-	Message         string
-	ShowProjectName bool
-	ShowReleaseName bool
-	ShowChangelog   bool
-	ShowDeployments bool
-	ShowSourceCode  bool
+	Message          string
+	ShowProjectName  bool
+	ShowReleaseTitle bool
+	ShowReleaseNotes bool
```

Comment:
Fields in `ReleaseNotificationConfig` were renamed to match fields in Release model.